### PR TITLE
Added RouteAcquisition File & Downloaded all Data

### PR DESCRIPTION
**Pull request by dodedic**
Iterates over all airports and months of the year, data stored on Sharepoint.
Added .env to .gitignore file.

@michaelweinold for some reason I can't access they key from the .env file, am still working on it.

### CODE REVIEW

**Comment by dodedic** [comment]
@michaelweinold both older commits 30f20ee and ca8a3c8 contain sensitive data from a week ago, we need to remove those first.

**Comment by michaelweinold** [comment]
> we need to remove those first.

Be my guest 😎

File: ADB/Airport Data/RouteAcquisition.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import os
+import requests
+import json
+from dotenv import load_dotenv
```
Comment (michaelweinold) [inline ADB/Airport Data/RouteAcquisition.py:4]: I would recommend we use native Python functionality instead. [TOML support is relatively new](https://realpython.com/python-toml/). You can take some inspiration from: https://github.com/sustainableaviation/EcoPyLot/blob/8a6b7b5dff270293a38754459cb5f2812471245e/ecopylot/utils.py#L6

File: ADB/Airport Data/RouteAcquisition.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+
+load_dotenv()
+api_key = os.getenv('ADB_KEY')
+
+# Function to retrieve data from the API for a given airport ICAO code and date
```
Comment (michaelweinold) [inline ADB/Airport Data/RouteAcquisition.py:11]: All fine for a draft PR. When we go to production, I suggest we follow [the NumPy docstrings style guide](https://numpydoc.readthedocs.io/en/latest/format.html).

File: ADB/Healtcheck.py
Comment (michaelweinold) [inline ADB/Healtcheck.py:1]: This could later be moved to a single function - with some docstring documentation.

**Comment by michaelweinold** [comment]
> I now just simply reset the compromised API key

hacker-hackerman

File: ADB/RouteAcquisition - Kopie.py
Comment (michaelweinold) [inline ADB/RouteAcquisition - Kopie.py:1]: Is this file an undesired copy?

**Review by michaelweinold — CHANGES_REQUESTED** [review]
Nice work - see my comments.

**Comment by dodedic** [comment]
@michaelweinold Changes implemented as per comments.

**Comment by michaelweinold** [comment]
...some more work needed, @dodedic.

**Comment by dodedic** [comment]
I cannot see any new comments though @michaelweinold

**Review by michaelweinold — CHANGES_REQUESTED** [review]
...can you see my comments now?

**Comment by dodedic** [comment]
All done! @michaelweinold 
Also added the config.toml file template now as well and changed the .gitignore back to ignore it!

File: ADB/AirportData/RouteAcquisition.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import os
+import requests
+import json
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+import toml
```
Comment (michaelweinold) [inline ADB/AirportData/RouteAcquisition.py:6]: Excellent - just native imports!

File: ADB/AirportData/config.toml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[api]
+key = "API_KEY_HERE"
```
Comment (michaelweinold) [inline ADB/AirportData/config.toml:2]: Nice, nice, nice!

File: ADB/Healtcheck.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import os
+import requests
+import toml
+
+
+def get_api_key():
```
Comment (michaelweinold) [inline ADB/Healtcheck.py:6]: As I mentioned earlier (https://github.com/sustainableaviation/demandmap/pull/20#discussion_r1584561530), this is all fine for now. Before we release the code, you will need to add Numpy-style doctrings.

**Pull request merged**